core/src/main/java/eu/krzdabrowski/starter/core/navigation/NavigationManager.kt

```diff
@@ -20,4 +20,14 @@ class NavigationManager @Inject constructor(
             navigationCommandChannel.send(command)
         }
     }
+
+    fun back() {
```

Comment:
I'd suggest using a verb for methods' names so something like `goBack` would be better 🙂

Reply:
hmm I think that's a great idea, let me add the fix